examples/flax/README.md

```diff
@@ -0,0 +1,70 @@
+
+*NOTE*: Currently, there is no "Trainer" abstraction for JAX/Flax -- all examples contain an explicit training loop.
+
+## Intro: JAX and Flax
+
+[JAX](https://github.com/google/jax) is a numerical computation library that exposes a NumPy-like API with tracing capabilities. With JAX's `jit`, you can
+trace pure functions and compile them into efficient, fused accelerator code on both GPU and TPU. JAX
+supports additional transformations such as `grad` (for arbitrary gradients), `pmap` (for parallelizing computation on multiple devices), `remat` (for gradient checkpointing), `vmap` (automatic
+efficient vectorization), and `pjit` (for automatically sharded model parallelism). All JAX transformations compose arbitrarily with each other -- e.g., efficiently
+computing per-example gradients is simply `vmap(grad(f))`.
+
+[Flax](https://github.com/google/flax) builds on top of JAX with an ergonomic
+module abstraction using Python dataclasses that leads to concise and explicit code. Flax's "lifted" JAX transformations (e.g. `vmap`, `remat`) allow you to nest JAX transformation and modules in any way you wish. Flax is the most widely used JAX library, with [129 dependent projects](https://github.com/google/flax/network/dependents?package_id=UGFja2FnZS01MjEyMjA2MA%3D%3D) as of May 2021. It is also the library underlying all of the official Cloud TPU JAX examples. (TODO: Add link once it's there.)
+
+## Running on Cloud TPU
+
+All of our JAX/Flax models are designed to run efficiently on Google
+Cloud TPUs. Here is a guide for running jobs on Google Cloud TPU.
+(TODO: Add a link to the Cloud TPU JAX getting started guide once it's public)
+Each example README contains more details on the specific model and training
+procedure.
+
+## Supported models
+
+Porting models from PyTorch to JAX/Flax is an ongoing effort. 
+If you are interested in contributing a model in JAX/Flax, please have a look at this 
+[Porting a PyTorch model to Flax (TODO: Patrick)]() guide and feel free to contact the
+🤗 Hugging Face team.
+
+Currently, the following models are supported in Flax:
+
+* [BERT](../../src/transformers/models/bert/modeling_flax_bert.py)
+* [RoBERTa](../../src/transformers/models/roberta/modeling_flax_roberta.py)
+* [GPT2](../../src/transformers/models/gpt2/modeling_flax_gpt2.py)
+* [ELECTRA](../../src/transformers/models/electra/modeling_flax_electra.py)
+
+For a complete overview of models that are supported in JAX/Flax, please have a look at [this (TODO: Patrick - wait for PR: #11758 to be merged]( ) table.
+
+Over 3000 pretrained checkpoints are supported in JAX/Flax as of May 2021.
+Click [here (TODO: will be enabled in a couple of days](https://huggingface.co/models?filter=jax) to see the full list on the 🤗 hub. 
```

Comment:
We are currently working on adding a "Jax" tag to the hub to easily filter all models that have uploaded jax weights in the same way it's done for PyTorch: https://huggingface.co/models?filter=pytorch